__tests__/components/common/Row.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react-native';
+import React from 'react';
+
+import { View } from 'react-native';
+import Row from '../../../components/common/Row';
+
+describe('<Row />', () => {
+  const children = <View />;
+
+  it('renders correctly', () => {
+    render(<Row children={children} />);
+
+    screen.debug();
```

Comment:
Can we remove these `screen.debug();` statements across all the tests? They are good for debugging, but after that they are only polluting the test output.

Reply:
cool, I forgot to remove them